### PR TITLE
dust: new, 1.1.1

### PR DESCRIPTION
**Pull request by purofle**
Topic Description
-----------------

- dust: new, 1.1.1

Package(s) Affected
-------------------

- dust: new, 1.1.1

Security Update?
----------------

No

Build Order
-----------

```
#buildit dust
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

Architectural progress for secondary ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-utils/dust/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,6 @@
+PKGNAME=dust
+PKGSEC=utils
+BUILDDEP="llvm rustc"
+PKGDES="A more intuitive version of du in rust"
```
Comment (eatradish) [inline app-utils/dust/autobuild/defines]:
```
saki@MagMilk [ oma@master ] $ ldd /home/saki/.cargo/bin/dust
        linux-vdso.so.1 (0x00007fbc39920000)
        libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007fbc395d6000)
        libc.so.6 => /usr/lib/libc.so.6 (0x00007fbc39200000)
        /lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007fbc39922000)
```

Add `PKGDEP="glibc gcc-runtime"`

**Review by eatradish — CHANGES_REQUESTED** [review]

File: app-utils/dust/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+ABSPLITDBG=0
```
Comment (eatradish) [inline app-utils/dust/autobuild/defines]: Why `ABSPLITDBG=0` ?

**Review by eatradish — CHANGES_REQUESTED** [review]

**Comment by eatradish** [comment]
LGTM on AMD64:

图片

**Comment by eatradish** [comment]
Also LGTM in AArch64 and LoongArch64

**Review by eatradish — DISMISSED** [review]

File: app-utils/dust/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,10 @@
+PKGNAME=dust
+PKGSEC=utils
+BUILDDEP="llvm rustc"
+PKGDEP="glibc gcc-runtime"
+PKGDES="A more intuitive version of du in rust"
```
Comment (MingcongBai) [inline app-utils/dust/autobuild/defines]: A TUI disk usage analyzer

File: app-utils/dust/autobuild/defines [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+USECLANG=1
+
+# FIXME: ld.lld is not support loongson3
```
Comment (MingcongBai) [inline app-utils/dust/autobuild/defines]: FIXME: ld.lld is not yet available.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**